spec/models/venue_spec.rb

```diff
@@ -18,5 +18,10 @@
     venue.location = nil
     expect(venue).to_not be_valid
   end
-  
+
+  it "can have many events" do
+    venue.events.create(title: "The Hobbit", description: "Hobbit movie 3")
+    expect(venue.events.map(&:venue_id)).to eq([venue.id])
```

Comment:
I like that you used venue.events.create but in this case the venue still just has one event. Is there a way you can add in another event and then assert that venue.events.count is 2?